### PR TITLE
Improve report button.

### PR DESCRIPTION
**Pull request by BigBlueHat**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "eleventy",
     "lint": "eslint .",
     "serve": "rm -rf _site/ && eleventy --serve",
+    "start": "rm -rf _site/ && eleventy --serve",
```
Comment (davidlehn) [inline package.json:10]: Are both of these needed?

**Comment by msporny** [comment]
Can we get rid of the "repo" button? The report itself points to the repo (or should?)

Can we rename the "report" button to "W3C VC Working Group Report"?

File: package.json (hunk 1 above)
Comment (BigBlueHat) [reply package.json:10]: I historically use `npm run serve` which is popular in the static site community. We use `npm start` here a lot. This accommodates both. Is there harm in having them both?

**Comment by BigBlueHat** [comment]
> Can we get rid of the "repo" button? The report itself points to the repo (or should?)
Repo is there to encourage use by implementers.
 
> Can we rename the "report" button to "W3C VC Working Group Report"?
Yes, but it's not always a "VC WG" report, but I'll see if I can get the correct group information into the button.

**Comment by BigBlueHat** [comment]
So...the `group` fields are wrong on all/most of the test suites. 🤦 ...so...gonna fix those, and then add some group lookups...

However, this feels like more trouble than it's worth and would prefer we ship this--which is an improvement--rather than waiting on perfection.

**Comment by jameseaster** [comment]
@BigBlueHat 
It looks like the "report" button is not always a report...
See _https://canivc.com/reports/vc-api-verifier-test-suite/_

We are differentiating between WG and CCG on the home page, could we use that same flag to change the title of the report button to dynamically render "W3C VC Working Group Report" vs "W3C VC CCG Report"?

I do think "Report" is better and more clear than just showing the url (what we are currently doing) as far as a UI aesthetic.

I also like the "repo" button as it provides easy access to the code they're being tested against. The report does point to the repo under "[Feedback](https://w3c.github.io/vc-data-model-2.0-test-suite/#:~:text=Feedback%3A-,GitHub,-w3c/vc%2Ddata)", but it doesn't seem as obvious to me as the "repo" button we have on canivc.com. Could we change it to "Test Suite Repo", to be more clear

Also the rest of the site has capitalized buttons, which would be nice to stay in theme with.

**Comment by BigBlueHat** [comment]
> It looks like the "report" button is not always a report... See https://canivc.com/reports/vc-api-verifier-test-suite/

That's part of a report. So...same concept, no?

> We are differentiating between WG and CCG on the home page, could we use that same flag to change the title of the report button to dynamically render "W3C VC Working Group Report" vs "W3C VC CCG Report"?

Yes...but only if we can get that data...which is currently broken somewhere in the food chain that generates the JSON we're using to get that link.

> Could we change it to "Test Suite Repo", to be more clear

Easy enough.

> Also the rest of the site has capitalized buttons, which would be nice to stay in theme with.

Sure.

File: package.json (hunk 1 above)
Comment (davidlehn) [reply package.json:10]: A DRY issue.  Could call one from the other too.  It's one line, it's fine.
A DRY issue.  Could call one from the other too.  It's one line, it's fine.

**Comment by BigBlueHat** [comment]
@jameseaster thought?
image

**Review by tminard — APPROVED** [review]

**Comment by jameseaster** [comment]
@BigBlueHat 
This looks like specs/ docs to me: https://w3c-ccg.github.io/vc-api/#verify-credential

That's what you're linked to when you click on https://canivc.com/reports/vc-api-verifier-test-suite/ and then click on the "report" button. 

I thought it would be more like this for a report:
https://w3c.github.io/vc-di-ecdsa-test-suite/#Data%20Integrity%20(ecdsa-rdfc-2019%20issuers)

**Comment by jameseaster** [comment]
Just wanting to note that in case there is a bug^

I might not know what "report" means in this case

**Review by jameseaster — APPROVED** [review]

**Comment by BigBlueHat** [comment]
> Just wanting to note that in case there is a bug^

That's certainly a bug/issue with the upstream metadata for that test suite.

This line needs changing:
https://github.com/w3c-ccg/vc-api-verifier-test-suite/blob/main/respecConfig.json#L6
which I'll do shortly.

Good find!

**Pull request merged**